### DISCUSSION TITLE
File portfolio

### DISCUSSION THREAD

**Discussion by ViralAgency** [docker/extension-ideas#190] [Ideas]
A simple file manager portfolio with tag profiling and descriptions for personal dockerfiles and docker compose files.

**Comment by Khaos66** [comment]
I was betting on it that Docker Desktop would manage compose files by now... AAAND I lost the bet =D

Anyhow. We have No-Pro Users that need to update their containers from time to time. Pulling new image versions is fine via the UI, but running `docker compose down` and `docker compose up -d` is too low level. Would be nice if a compose collection of containers at least had a `recreate` action or smth.
